tests/server/test_server.py

```diff
@@ -17,6 +17,7 @@ def __init__(self) -> None:
             tuple[str, str, str, float, float]
         ] = []  # uuid, email, pwd_hash, usd, btc
         self._db_tokens: list[tuple[str, str]] = []
+        self._db_prices: list[tuple[str, str]] = [("3.0", "01-01-2019")]  # price, date
```

Comment:
ElephantSQL actually stores exchange rate as floats, so it would make sense to leave it here as `[float, str]`. The property `db_prices` will have to use a similar list comprehension as in `db_users` property to convert floats into `str` (to match real database).